File: test/function.cpp
#include <ostream>
int function(int a);

float returnTypeInferenceFunction(int a);

int returnTypeInferenceFunction(auto a);

int function(int a)
{
  int b = 10;
  return a * b;
}

float returnTypeInferenceFunction(int a)
{
  float b = 10.2;
  return a * b;
}

int returnTypeInferenceFunction(auto a)
{
  float b = 10.2;
}

File: test/function.py
@transpile
def function(a:int)->int:
    b:int=10
    return a*b
def voidFunction(a:int):
    b:float=10.2
    c=a*b
@transpile
def returnTypeInferenceFunction(a:int):
    b:float=10.2
    return a*b
@transpile
def returnTypeInferenceFunction(a)->int:
    b:float=10.2

@transpile
def voidFunction(a:int):
    b:float=10.2
    c=a*b
